### DISCUSSION TITLE
Scrapli Appreciation

### DISCUSSION THREAD

**Discussion by mdetrick** [scrapli/scrapli_community#77] [General]
I want to say how much I appreciate Scrapli. I spent weeks trying to get Paramiko and Netmiko working with the Cisco MDS NXOS platform and they just do not work. It took me a couple of hours to dev python code with Scrapli and everything works great. Thank you!

**Comment by carlmontanari** [comment]
Hey @mdetrick Thanks a ton for the note, and very glad to have ya in the scrapli family! I'm glad scrapli is working well for ya!
